SplunkforPaloAltoNetworks/default/data/ui/views/iot_security.xml

```diff
@@ -1,14 +1,15 @@
 <form>
   <label>IoT Security</label>
   <fieldset autoRun="true" submitButton="true">
-    <input type="time" searchWhenChanged="true">
+    <input type="time" searchWhenChanged="true" token="time">
```

Comment:
Our other dashboards don't have a token for time which I think makes it apply globally. Did Splunk add this?